src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -43,8 +36,10 @@ public AddCommand parse(String args) throws ParseException {
         Email email = ParserUtil.parseEmail(argMultimap.getValue(PREFIX_EMAIL).get());
         Address address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get());
         Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
+         */
 
-        Person person = new Person(name, phone, email, address, tagList);
+        Person person = new Person(new Name("Placeholder"), new HashMap<>(), new HashMap<>(), new HashMap<>(),
+                new Company("Placeholder"), new JobTitle("Placeholder"), new Pronoun("Placeholder"), new HashMap<>());
```

Comment:
We might have to change the pronouns argument to a list of pronouns instead of a single aggregated string